lib/components/Auth/AuthService.ts

```diff
@@ -167,10 +166,58 @@ export function passwordIsInListOfMostCommonPasswords(password): boolean {
 
 
 
+export function decodeJWTToken(token) {
+    let decodedToken;
+
+    try {
+        decodedToken = jwt.verify(token, CONSTANTS.JWT_SECRET);
+    } catch (error) {
+        throw new Error('invalid token');
+    }
+
+    return decodedToken;
+}
+
+
+
+export function verifyJWTToken(decodedToken, options) {
```

Comment:
Verifies that the token attached to the user's session is decoded and storing the expected user_id and expiration timestamp